### PR TITLE
apriltag subsystem

### PR DESCRIPTION
**Pull request by AayushK123**
it is a new apriltag subsystem to get networktable entries

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/AprilTagSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    tableLeft = NetworkTableInstance.getDefault().getTable("apriltag").getSubTable("left");
+    this.tagIdLeftCam = tableLeft.getEntry("tagId");
+    this.yRotationLeftCam = tableLeft.getEntry("angleY");
+    this.horizontalOffsetLeftCam = tableLeft.getEntry("horizontalOffset");
+    this.realDistanceLeftCam = tableLeft.getEntry("realDistance");
+
+    tableRight = NetworkTableInstance.getDefault().getTable("apriltag").getSubTable("right");
+    this.tagIdRightCam = tableRight.getEntry("tagId");
+    this.horizontalOffsetRightCam = tableRight.getEntry("horizontalOffset");
```
Comment (bengold22) [inline src/main/java/frc/robot/subsystems/AprilTagSubsystem.java]: Why don't you have an angle for the right camera?

File: src/main/java/frc/robot/subsystems/AprilTagSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    this.realDistanceRightCam = tableRight.getEntry("realDistance");
+
+  }
+
+  public Long getTagId() {
+    return tagIdLeftCam.getInteger(0L);
```
Comment (bengold22) [inline src/main/java/frc/robot/subsystems/AprilTagSubsystem.java]: This logic should be:

get ID of left cam
if not null, return left cam ID
else 
return right cam ID

File: src/main/java/frc/robot/subsystems/AprilTagSubsystem.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  }
+
+  public Double getRotation() {
+    double rotation = yRotationLeftCam.getDouble(0.0) + yRotationRightCam.getDouble(0.0);
```
Comment (bengold22) [inline src/main/java/frc/robot/subsystems/AprilTagSubsystem.java]: As discussed, this won't work

**Review by bengold22 — CHANGES_REQUESTED** [review]

**Review by MatthewSpofford — APPROVED** [review]
Functionally it looks good to me (and all of @bengold22's comments appear to be resolved). My one nitpicky suggestion if you have extra time, is think of a way to generalize and remove the repeating if/else statements that are checking the camera IDs.

**Review by bengold22 — APPROVED** [review]

**Pull request merged**